### PR TITLE
Lightning Talk data and tool

### PR DESCRIPTION
**Pull request by merriam**
Add lightning talk descriptions for all sessions of  SciPy 2017 and PyCon US 2017.  Added a spreadsheet of the data and tool to parse it into descriptions to tools/

### CODE REVIEW

File: tools/parse_lightning_spreadsheet.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
+import openpyxl as op
+from textwrap import wrap
```
[the remark this answers is not in the document]
Comment (merriam) [reply tools/parse_lightning_spreadsheet.py]: Not exactly true.

The RST will fail to parse unless the tables align exactly.   I could make the table very, very wide.  This makes it not human debuggable.   

Unfortunately, rendering RST natively on OSX requires a stack of tooling, so I check over the table in text form.

File: tools/parse_lightning_spreadsheet.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    basic_line_format = '+ {:' + str(width[0]) + '} | {:' + str(width[1]) + '} | {:' + str(width[2]) + '} +'
+
+    lines = table_header_part(topic_name)
+    for talk in talks:
+        talk_lines = table_part_for_talk(talk)
+        lines.extend(talk_lines)
+        lines.append(sep_line)
+    return "\n".join(lines)
+
+def quote_for_json(unquoted_new_contents):
```
[the remark this answers is not in the document]
Comment (merriam) [reply tools/parse_lightning_spreadsheet.py]: Someday, not in our lifetimes, quoting will actually work correctly.

[the remark this answers is not in the document]
Comment (merriam) [reply tools/parse_lightning_spreadsheet.py]: OK, I'll toss another pass in.  Rendered fine in my viewer. :<

File: tools/parse_lightning_spreadsheet.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
+import openpyxl as op
```
[the remark this answers is not in the document]
Comment (merriam) [reply tools/parse_lightning_spreadsheet.py]: OK, I can pull out the spreadsheet and just upload the regular files.   Having to save every change as a CSV file would be a pain, and also I would need to dump the current code and rewrite.

File: tools/parse_lightning_spreadsheet.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Daniel-at-github) [reply tools/parse_lightning_spreadsheet.py]: This is how I render it:
~~~ bash
cd ~/git/pyvideo
make html VERBOSE=1 && make serve
~~~
If it works in OSX, it give you a copy of the web (this is how it's made).

File: tools/parse_lightning_spreadsheet.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Daniel-at-github) [reply tools/parse_lightning_spreadsheet.py]: OK, it's useful then.

File: tools/parse_lightning_spreadsheet.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (Daniel-at-github) [reply tools/parse_lightning_spreadsheet.py]: I don't feel that I can merge a pull request with an `.xlsx`, it goes against the spirit of pyvideo/data. Everything here is in json, an open versionable format that every one can read.

`.xlsx` format is crafted to be easy and smooth in some spreadsheets, but in others spreadsheets it's designed to be painful ("almost" works, the standard changes over time to maintain incompatibility, ...).
As second option `.xlsx` it's OK, as only option not so.

But I'm only one collaborator of the repo, ask to the others for a review or chat about this in gitter for more opinions.

**Comment by merriam** [comment]
Unfortunately, git deals badly with 'requested changes' so I'll reopen this.

**Pull request closed without merging**